index.js

```diff
@@ -6,30 +6,16 @@ var xtend = require("xtend")
 
 module.exports = createXHR
 createXHR.XMLHttpRequest = window.XMLHttpRequest || noop
-createXHR.XDomainRequest = "withCredentials" in (new createXHR.XMLHttpRequest()) ? createXHR.XMLHttpRequest : window.XDomainRequest
 createXHR.qsSerialize = null // Define this as a function to support the `qs` option
 
-forEachArray(["get", "put", "post", "patch", "head", "delete"], function(method) {
+"get,put,post,patch,head,delete".split(",").map(function(method) {
```

Comment:
Why make a string and split instead of just creating an array? `["get","put","post","patch","head","delete"].map()`